src/components/DiagnosticsBody.vue

```diff
@@ -83,7 +84,7 @@ export default Vue.extend({
       this.$store.dispatch('diagnostics/updateDiagnostic', { isMuted, row });
     },
     toggleMute() {
-      this.showMuted = !this.showMuted;
+      this.$store.dispatch('preferences/toggleShowMuted');
```

Comment:
Hmm, from a user point of view, shouldn't this just be set to whatever the current value of the toggle switch is, instead of toggling? (In case the pref was changed in the background and the front end wasn't updated somehow.)